src/Adapter/Module/Module.php

```diff
@@ -349,7 +351,7 @@ public function onReset()
             return false;
         }
 
-        return $this->instance->reset();
+        return method_exists($this->instance, 'reset') ? $this->instance->reset() : true;
```

Comment:
Watch out https://www.php.net/manual/en/function.method-exists.php#124462

Reply:
Interesting and fixed ;) Thanks :heart_eyes: